src/main/webapp/WEB-INF/tags/footer.tag

```diff
@@ -25,7 +41,7 @@
             </div>
             <div>
                 <h6 class="grey-text dev-statement center-align text-lighten-2">
-                    <b>
+                    <br>
```

Comment:
Is that change from <b> to <br> supposed to be there?

Reply:
no, thank you, that should not be there